UnityProject/Assets/Scripts/Weapons/Meleeable.cs

```diff
@@ -7,6 +7,19 @@
 /// </summary>
 public class Meleeable : MonoBehaviour, ICheckedInteractable<PositionalHandApply>
 {
+	[SerializeField]
+	private ItemTrait butcherKnifeTrait;
+
+	[SerializeField]
```

Comment:
@ziiplock no need to put SerializeField on static fields. They can't be changed in editor.